### DISCUSSION TITLE
Map name

### DISCUSSION THREAD

**Discussion by FrogMan241** [xanthics/poe_gen_gwennen#19] [Ideas]
Hi, could you perhaps create a version of the regex for map names? I am running a strat to rush the bosses in my maps, but want to horizon bad maps with long layouts. I know you have the code available to generate the ngrams, but I'm not a python coder and I don't think it should be too difficult to generate them for the map names. Please and thank you.

**Comment by xanthics** [comment]
I had a project for map mods, but veiset has taken over that niche along with leveling vendor strings.  I'd suggest checking with him first at https://github.com/veiset/poe-vendor-string
